### ISSUE TITLE
sudo make install fails - it's looking for License.txt

### ISSUE THREAD

**Issue by devangel77b** [mdrohmann/mtocpp#5]
Building from source on Ubuntu 16.04 is successful (with some warnings about Doxygen fonts shipped) but when I try to execute sudo make install it fails looking for License.txt
```
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/bin/mtocpp
-- Installing: /usr/local/bin/mtocpp_post
CMake Error at cmake_install.cmake:76 (file):
  file INSTALL cannot find "/home/evangeli/Downloads/mtocpp/License.txt".


Makefile:82: recipe for target 'install' failed
make: *** [install] Error 1
```
The problem goes away if I copy LICENSE to License.txt; so probably somewhere there is a config file with the wrong filename listed (LICENSE vs License.txt)? I looked but couldn't find it easily.

**Comment by nunoguedelha** [comment]
Hi @devangel77b , I just installed this tool and run onto the same issue. The problem is in the `CMakeLists.txt`. More precisely, in this section: https://github.com/mdrohmann/mtocpp/blob/8b178cbf0f43bfad187a64f68d5fbe6011126a69/CMakeLists.txt#L97

the `License.txt` file is copied from `LICENSE` only if `WS_SRC` is defined, which is not the case by default, so `cmake` execution fails later in this line: https://github.com/mdrohmann/mtocpp/blob/8b178cbf0f43bfad187a64f68d5fbe6011126a69/CMakeLists.txt#L136
I just did a PR for fixing the issue.
